registry/DeploymentTemplates/azuredeploy.parameters.json

```diff
@@ -48,10 +48,10 @@
             "value": ""
         },
         "registryTag": {
-            "value": ""
+            "value": "2.7.1"

```

Comment:
any reason for keeping `REGISTRY_TAG` around?

Reply:
This is interim to keep CI code happy.